src/reducers/entities/posts.js

```diff
@@ -90,6 +90,35 @@ function handleReceivedPosts(posts = {}, postsInChannel = {}, action) {
     return {posts: nextPosts, postsInChannel: nextPostsForChannel};
 }
 
+function handlePostsFromSearch(posts = {}, postsInChannel = {}, action) {
+    const newPosts = action.data.posts;
+    let info = {posts, postsInChannel};
+    const postsForChannel = new Map();
+
+    for (const id in newPosts) {
+        if (newPosts.hasOwnProperty(id)) {
+            const nextPost = newPosts[id];
+            if (nextPost) {
```

Comment:
Why are we checking if this is falsey? Shouldn't we know it exists because we just `id` is a key in `newPosts`?